code/lib/blocks/package.json

```diff
@@ -54,7 +54,7 @@
     "@types/color-convert": "^2.0.0",
     "color-convert": "^2.0.1",
     "es-toolkit": "^1.22.0",
-    "markdown-to-jsx": "^7.4.5",
+    "markdown-to-jsx": "7.7.2",
```

Comment:
Does this have to be fixed? Can we use `^7.7.2` instead?

Reply:
Whoops, sorry, fat fingers